catalyst-gateway/bin/src/service/api/cardano/staking/assets_get.rs

```diff
@@ -100,8 +97,8 @@ pub(crate) async fn endpoint(
     .into()
 }
 
-/// TXO asset information.
-struct TxoAssetInfo {
+/// TXO Native Token information.
+struct NativeTokens {
```

Comment:
Bad name, they are defined as "Assets" in the ledger spec and thats what we should call them.  Native Token never appears in the ledger specification at all.  We should track the official language in the specification.
See: https://github.com/IntersectMBO/cardano-ledger/blob/8467e3585304d0a69a39232178db114a52d25323/eras/conway/impl/cddl-files/conway.cddl#L72

Revert the name change.